### DISCUSSION TITLE
Welcome to node-yahoo-finance2 Discussions!

### DISCUSSION THREAD

**Discussion by gadicc** [gadicc/yahoo-finance2#119] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ZehCoque** [comment]
Hello! 

First of all, ***congratulations for the awesome work!*** This is a very good API! 

I was wondering if it is possible to get the split history, the same way the Yahoo! charts show:

![image](https://user-images.githubusercontent.com/37452468/125376329-2199ea00-e361-11eb-9c5c-fa76ba6d9f5e.png)
![image](https://user-images.githubusercontent.com/37452468/125380508-a9372700-e368-11eb-898d-99d4a665c883.png)

I am sure this print can help:

![image](https://user-images.githubusercontent.com/37452468/125380818-367a7b80-e369-11eb-99f4-612d8aaf9730.png)

if you change the "Show" option to Stock Splits, the exported csv shows the information I need.

Looking forward to your reply!
Thanks!

**Reply by ZehCoque** [reply]
I managed to make it work like this:

```
const queryOptions = { period1: '2021-06-30', events: 'history split',};
yahooFinance.historical('BPAC11.SA', queryOptions, { validateResult: false })
```

with this result:

`[ { date: 2021-07-01T00:00:00.000Z, stockSplits: '4:1' } ]`

But this clearly is not the way this API is supposed to work. In this way, I am only able to get the stockSplits response and I would like to get everything (high, low, open, close, volume *AND* splits) at once. Is it possible?

Thanks!

**Comment by fan777** [comment]
Just wanted to say hi and thank you for creating this! I have used it extensively for my final capstone project in a coding bootcamp. Thanks!

**Reply by gadicc** [reply]
Thanks so much, @fan777!  It's really great to get messages like this.  We're really happy it helped you with your project!
